cms/models.py

```diff
@@ -38,7 +38,7 @@ class CourseProgramChildPage(Page):
     class Meta:
         abstract = True
 
-    parent_page_types = ["CoursePage", "ProgramPage"]
+    parent_page_types = ["CoursePage", "ProgramPage", "HomePage"]
```

Comment:
how much work would require if we rename `CourseProgramChildPage` model name to generic name?

Reply:
Overall (because renaming would pretty much go for every section we developed for the course page) that would require updating our models, in turn requiring migrations. Since the models change and the tables change, we would perhaps have to rename the attributes accordingly as well, because the makes sense. Going forward that would mean we will have to update our tests and templates as well.

If we have time to do that, I would *strongly* advise renaming these classes to their generic names right now, while we can. Later on this will turn into a tech debt that will be blocked with months of technical development dependent on it and this won't practically be feasible at that point. Getting this done right now would mean we keep the base clean from the start.

@pdpinch your thoughts on this?

Reply:
Until we have a working home page, I can't justify the time on this tech debt.